Traceback (most recent call last):
  File "./probing.py", line 55, in <module>
    _, _, embedding_dimension = X.shape
ValueError: too many values to unpack (expected 3)
Traceback (most recent call last):
  File "./probing.py", line 55, in <module>
    _, _, embedding_dimension = X.shape
ValueError: too many values to unpack (expected 3)
Traceback (most recent call last):
  File "./probing.py", line 55, in <module>
    _, _, embedding_dimension = X.shape
ValueError: too many values to unpack (expected 3)
Traceback (most recent call last):
  File "./probing.py", line 30, in <module>
    X = np.load(
  File "/home/zw545/.local/lib/python3.8/site-packages/numpy/lib/npyio.py", line 390, in load
    fid = stack.enter_context(open(os_fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: '../X/X8X_raw_layer_6_bert-uncased.npy'
Traceback (most recent call last):
  File "./probing.py", line 30, in <module>
    X = np.load(
  File "/home/zw545/.local/lib/python3.8/site-packages/numpy/lib/npyio.py", line 390, in load
    fid = stack.enter_context(open(os_fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: '../X/X8X_raw_layer_6_bert-uncased.npy'
Traceback (most recent call last):
  File "./probing.py", line 30, in <module>
    X = np.load(
  File "/home/zw545/.local/lib/python3.8/site-packages/numpy/lib/npyio.py", line 390, in load
    fid = stack.enter_context(open(os_fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: '../X/X8X_raw_layer_6_bert-uncased.npy'
Traceback (most recent call last):
  File "./probing.py", line 55, in <module>
    _, _, embedding_dimension = X.shape
ValueError: too many values to unpack (expected 3)
Traceback (most recent call last):
  File "./probing.py", line 55, in <module>
    _, _, embedding_dimension = X.shape
ValueError: too many values to unpack (expected 3)
Traceback (most recent call last):
  File "./probing.py", line 55, in <module>
    _, _, embedding_dimension = X.shape
ValueError: too many values to unpack (expected 3)
Traceback (most recent call last):
  File "./probing.py", line 30, in <module>
    X = np.load(
  File "/home/zw545/.local/lib/python3.8/site-packages/numpy/lib/npyio.py", line 390, in load
    fid = stack.enter_context(open(os_fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: '../X/X8X_raw_layer_6_bert-uncased.npy'
Traceback (most recent call last):
  File "./probing.py", line 30, in <module>
    X = np.load(
  File "/home/zw545/.local/lib/python3.8/site-packages/numpy/lib/npyio.py", line 390, in load
    fid = stack.enter_context(open(os_fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: '../X/X8X_raw_layer_6_bert-uncased.npy'
Traceback (most recent call last):
  File "./probing.py", line 30, in <module>
    X = np.load(
  File "/home/zw545/.local/lib/python3.8/site-packages/numpy/lib/npyio.py", line 390, in load
    fid = stack.enter_context(open(os_fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: '../X/X8X_raw_layer_6_bert-uncased.npy'
Traceback (most recent call last):
  File "./probing.py", line 55, in <module>
    _, _, embedding_dimension = X.shape
ValueError: too many values to unpack (expected 3)
Traceback (most recent call last):
  File "./probing.py", line 55, in <module>
    _, _, embedding_dimension = X.shape
ValueError: too many values to unpack (expected 3)
Traceback (most recent call last):
  File "./probing.py", line 55, in <module>
    _, _, embedding_dimension = X.shape
ValueError: too many values to unpack (expected 3)
Traceback (most recent call last):
  File "./probing.py", line 30, in <module>
    X = np.load(
  File "/home/zw545/.local/lib/python3.8/site-packages/numpy/lib/npyio.py", line 390, in load
    fid = stack.enter_context(open(os_fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: '../X/X8X_raw_layer_6_bert-uncased.npy'
Traceback (most recent call last):
  File "./probing.py", line 30, in <module>
    X = np.load(
  File "/home/zw545/.local/lib/python3.8/site-packages/numpy/lib/npyio.py", line 390, in load
    fid = stack.enter_context(open(os_fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: '../X/X8X_raw_layer_6_bert-uncased.npy'
Traceback (most recent call last):
  File "./probing.py", line 30, in <module>
    X = np.load(
  File "/home/zw545/.local/lib/python3.8/site-packages/numpy/lib/npyio.py", line 390, in load
    fid = stack.enter_context(open(os_fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: '../X/X8X_raw_layer_6_bert-uncased.npy'
Traceback (most recent call last):
  File "./probing.py", line 55, in <module>
    _, _, embedding_dimension = X.shape
ValueError: too many values to unpack (expected 3)
Traceback (most recent call last):
  File "./probing.py", line 55, in <module>
    _, _, embedding_dimension = X.shape
ValueError: too many values to unpack (expected 3)
Traceback (most recent call last):
  File "./probing.py", line 55, in <module>
    _, _, embedding_dimension = X.shape
ValueError: too many values to unpack (expected 3)
Traceback (most recent call last):
  File "./probing.py", line 30, in <module>
    X = np.load(
  File "/home/zw545/.local/lib/python3.8/site-packages/numpy/lib/npyio.py", line 390, in load
    fid = stack.enter_context(open(os_fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: '../X/X8X_raw_layer_6_bert-uncased.npy'
Traceback (most recent call last):
  File "./probing.py", line 30, in <module>
    X = np.load(
  File "/home/zw545/.local/lib/python3.8/site-packages/numpy/lib/npyio.py", line 390, in load
    fid = stack.enter_context(open(os_fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: '../X/X8X_raw_layer_6_bert-uncased.npy'
Traceback (most recent call last):
  File "./probing.py", line 30, in <module>
    X = np.load(
  File "/home/zw545/.local/lib/python3.8/site-packages/numpy/lib/npyio.py", line 390, in load
    fid = stack.enter_context(open(os_fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: '../X/X8X_raw_layer_6_bert-uncased.npy'
